jekyll/_cci2/enable-checks.md

```diff
@@ -32,9 +31,11 @@ GitHub does not currently provide a granular way for you to rerun workflows. Whe
 GitHub Checks should not be confused with GitHub status updates:
 
 * GitHub Checks are administered from the GitHub UI, and are reported in the GitHub UI per _workflow_.
+  * GitHub Checks from OAuth integrations have the same name as the workflow.
+  * GitHub Checks from GitHub App integrations names are a combination of workflow name and the first eight characters of the pipeline definition ID. The identifier is used to prevent naming collisions. The pipeline definition ID is available in **Project Settings** > **Pipelines** page in the CircleCI app.
```

Comment:
```suggestion
  * GitHub Checks from OAuth integrations have the same name as the associated workflow.
  * GitHub Checks from GitHub App integrations are named sing a combination of the workflow name and the first eight characters of the pipeline definition ID. The identifier is used to prevent naming collisions. The pipeline definition ID is available in the **Project Settings** > **Pipelines** page in the CircleCI web app.
```